doc/sphinx/source/community/index.rst

```diff
@@ -35,3 +35,4 @@ help, e.g. on our
 		GitHub workflow <repository>
 		Moving work from the private to the public repository <private_repository>
 		Release schedule and procedure <release_strategy>
+                Detailed testing before release <detailed_release_procedure>
```

Comment:
How about including this as part of the release chapter instead of adding it as a separate page here? That keep related topics together a bit better

Reply:
I understand your point but I think both pages are very long already. I think merging these would make it hard to read

Reply:
How about adding it as a subpage of that then?